src/answers-umd.js

```diff
@@ -418,6 +418,61 @@ class Answers {
     this.coreAdapter.storage.setWithPersist(StorageKeys.QUERY, query);
   }
 
+  /**
+   * Conduct a vertical search with the provided options.
+   *
+   * @param {Object} options
+   * @param {string} options.verticalKey the vertical ID for the search
+   * @param {string} options.query the query string of the search
+   * @param {boolean} options.useFacets enables facets if true
+   * @param {boolean} options.resetPagination resets the search offset if true
+   * @param {string} options.queryId the queryId associated with the search
+   * @param {boolean} options.appendQuery if true, adds the results of this query to the current results
+   */
+  verticalSearch (options) {
+    const verticalKey = options.verticalKey;
+    const verticalOptions = {
+      ...options.useFacets && { useFacets: options.useFacets },
+      ...options.resetPagination && { resetPagination: options.resetPagination }
+    };
+    const queryOptions = {
+      ...options.query && { input: options.query },
+      ...options.queryId && { id: options.queryId },
+      ...options.appendQuery && { append: options.appendQuery }
+    };
+    this.coreAdapter.verticalSearch(verticalKey, verticalOptions, queryOptions);
+  }
+
+  /**
+   * Rests all filters in the filter registry.
```

Comment:
nit: Resets

Reply:
I am setting filters in the filter registry for my use case as well. Did we want to support CRUD for these filters as well? This is almost feeling like a wrapper around core 🤔

Reply:
I don't think a wrapper is necessarily a bad thing, because it lets us swap out the internals without changing the public interface. But you're right, we could just rename coreAdapter to core and access everything we need with ANSWERS.core or ANSWERS.core.storage

Reply:
@tmeyer2115 What are your thoughts on this?